src/Kliva/Controls/ActivityFeedControl.xaml.cs

```diff
@@ -68,5 +69,35 @@ private void ScrollToTop()
         {
             _scrollViewer?.ScrollToVerticalOffsetWithAnimation(0);
         }
+
+        private void ActivityList_SelectionChanged(object sender, SelectionChangedEventArgs e)
+        {
+            if (IsDesktopState())
+            {
+                // Only update the ViewModel if we are in the wide state where the list supports
+                // selection
+                ViewModel.SelectedActivity = (ActivitySummary)e.AddedItems.FirstOrDefault();
```

Comment:
Why do we need SelectionChanged in code to update the VM and SelectedItem binding to the VM on the control itself?